README.adoc

```diff
@@ -127,6 +127,19 @@ NOTE: Debug output from `vagrant-service-manager` is prepended with
 the following string:
 `DEBUG command: [ service-manager: <command name / log message> ]`
 
+=== Configurations
+
+- HTTP Proxy Settings +
+User can able to start ADB in proxy environment by specifying following configurations
+through Vagrantfile as:
+
++
+----------------------------------------------------------------------------
+config.servicemanager.http_proxy = <Proxy URL>
+config.servicemanager.http_proxy_user = <Proxy user name>
+config.servicemanager.http_proxy_password = <Proxy user password>
+----------------------------------------------------------------------------
+
```

Comment:
When these settings are applied, they are passed through to the Docker and OpenShift service. _http_proxy_user_ and _http_proxy_password can be omitted in an unauthenticated proxy environment